hal/LowPowerTickerWrapper.cpp

```diff
@@ -32,15 +32,7 @@ void LowPowerTickerWrapper::irq_handler(ticker_irq_handler_type handler)
 {
     core_util_critical_section_enter();
 
-    if (_suspended) {
-        if (handler) {
-            handler(&data);
```

Comment:
Even when suspended interrupts need to be passed through so higher layers can run tests.

Reply:
Please note the interrupts are passed as before. I moved `_suspended` to the next `if` statement to have the flags cleared (i.e. `_pending_match`).

Reply:
My bad, I didn't realize the interrupt was still getting through.